src/demos/select-field/select-field-demo.component.html

```diff
@@ -142,3 +142,44 @@ <h2 class="sky-section-heading">
     </div>
   </sky-column>
 </sky-row>
+
+<sky-row>
+  <sky-column screenSmall="12">
+    <h2 class="sky-section-heading">
+      Custom picker
+    </h2>
+
+    <div class="sky-form-group">
+      <label
+        class="sky-control-label"
+        id="my-select-field-custom-1">
```

Comment:
Code style > closing brackets need to be on their own line (there's a few in here).